bedrock/pocket/templates/pocket/includes/footer.html

```diff
@@ -8,94 +8,95 @@
                             rel="noopener noreferrer">
-                            <img src="{{ static('img/pocket/apple-app-store-badge.svg') }}" alt="Download On the Apple App Store">
+                            <img src="{{ static('img/pocket/apple-app-store-badge.svg') }}" alt="{{ ftl('pocket-footer-download-on-apple-app-store') }}">
                         </a>
                     </li>
                     <li>
                         <a class="app-download-link google-play-badge"
                             href="https://play.google.com/store/apps/details?id=com.ideashower.readitlater.pro"
                             target="_blank" rel="noopener noreferrer">
-                            <img src="{{ static('img/pocket/google-play-badge.png') }}" alt="Get It On Google Play">
+                            <img src="{{ static('img/pocket/google-play-badge.png') }}" alt="{{ ftl('pocket-footer-get-it-google-play') }}">
                         </a>
                     </li>
                 </ul>
             </section>
         </nav>
         <div class="pocket-footer-secondary">
           <div class="pocket-footer-legal-details">
-            <p class="pocket-footer-secondary-desc">Pocket is part of the <a href="https://www.mozilla.org/about/">Mozilla</a> family of products.</p>
-
-            <div class="legal" aria-label="Legal">
-              <span>© 2021 Read It Later, Inc.</span>
-              <a href="https://getpocket.com/privacy?src=footer_v2">Privacy policy</a>
-              <a href="https://getpocket.com/tos?src=footer_v2">Terms of service</a>
-              <button id="ot-sdk-btn" class="ot-sdk-show-settings">Cookie preferences</button>
+            <p class="pocket-footer-secondary-desc">
+            {{ ftl('pocket-footer-pocket-is-part-of-mozilla-family',
+            mozorgabout='https://www.mozilla.org/about/') }}</p>
+            <div class="legal" aria-label="{{ ftl('pocket-footer-aria-label-legal') }}">
+              <span>© 2021 {{ ftl ('pocket-footer-read-it-later-name') }}</span>
+              <a href="{{ url('pocket.privacy') }}?src=footer_v2">{{ ftl ('pocket-footer-privacy-policy') }}</a>
+              <a href="{{ url('pocket.tos') }}?src=footer_v2">{{ ftl ('pocket-footer-terms-of-service') }}</a>
+              <button id="ot-sdk-btn" class="ot-sdk-show-settings">{{ ftl ('pocket-footer-cookie-preferences') }}</button>
             </div>
           </div>
           <ul class="pocket-footer-socials">
             <li class="pocket-footer-socials-item">
-              <a href="https://twitter.com/pocket" class="pocket-footer-socials-item-link twitter" target="_blank" rel="external noopener noreferrer"><span>Twitter</span></a>
+              <a href="https://twitter.com/pocket" class="pocket-footer-socials-item-link twitter" target="_blank" rel="external noopener noreferrer"><span>{{ ftl ('-brand-name-twitter') }}</span></a>
```

Comment:
Issue (blocking): brand name variables aren't also Fluent strings that are usuable directly in the templates. Right now this renders as `<span>-brand-name-twitter</span>`. You'll need to create a new Fluent string ID for this text in an FTL file, and then reference the brand name there.

Reply:
Nice spot - thanks!